paddle/fluid/operators/linear_chain_crf_op.h

```diff
@@ -208,15 +208,17 @@ class LinearChainCRFGradOpKernel : public framework::OpKernel<T> {
     const T* ll_grad =
         ctx.Input<Tensor>(framework::GradVarName("LogLikelihood"))->data<T>();
 
-    Tensor* emission_grad =
-        ctx.Output<Tensor>(framework::GradVarName("Emission"));
+    auto* emission_grad =
+        ctx.Output<LoDTensor>(framework::GradVarName("Emission"));
     Tensor* transition_grad =
         ctx.Output<Tensor>(framework::GradVarName("Transition"));
 
     // TODO(caoying) Fix this constraint. When the Input(Emission) is from the
     // data reader operator, it can have no gradients.
     PADDLE_ENFORCE(emission_grad, "Output(Emission@Grad) should not be null.");
     emission_grad->mutable_data<T>(platform::CPUPlace());
+    emission_grad->set_lod(ctx.Input<LoDTensor>("Emission")->lod());
```

Comment:
Had better move `emission_grad->set_lod...` to `LinearChainCRFGradOp::InferShape`.